src/actions/secrets.js

```diff
@@ -57,13 +57,15 @@ export function deleteSecret(secrets, cancelMethod) {
     // from error handling and additional notifications on success/error
     const namespacesToSecretsMap = new Map();
     secrets.forEach(secret => {
-      let foundSecretInfo = namespacesToSecretsMap.get(secret.namespace);
+      let foundSecretInfo = namespacesToSecretsMap.get(
```

Comment:
For this and the lines below just pull out name and namespace at the top ?

Reply:
good call, done 👍